### PR TITLE
Device manager - tweak string formatting of default device name

### PR DESCRIPTION
**Pull request by kerryarchibald**
## Checklist

* [ ] Tests written for new code (and old code if feasible)
* [ ] Linter and other CI checks pass
* [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/vector-im/element-web/blob/develop/CONTRIBUTING.md))

---
Here's what your changelog entry will look like:

## ✨ Features
 * Device manager - tweak string formatting of default device name ([\#23457](https://github.com/vector-im/element-web/pull/23457)). Contributed by @kerryarchibald.

### CODE REVIEW

File: src/vector/platform/ElectronPlatform.tsx [hunk 1]
```diff
@@ -70,7 +69,7 @@ function platformFriendlyName(): string {
 function onAction(payload: ActionPayload): void {
     // Whitelist payload actions, no point sending most across
     if (['call_state'].includes(payload.action)) {
-        electron.send('app_onAction', payload);
+        window.electron.send('app_onAction', payload);
```
Comment (kerryarchibald) [inline src/vector/platform/ElectronPlatform.tsx:72]: this makes the testing mock work

File: src/vector/platform/IPCManager.ts [hunk 2]
```diff
@@ -60,6 +58,7 @@ export class IPCManager {
         }
 
         const callbacks = this.pendingIpcCalls[payload.id];
+
```
Comment (t3chguy) [inline src/vector/platform/IPCManager.ts:61]: any reason for this spurious extra line?

File: src/vector/platform/ElectronPlatform.tsx [hunk 3]
```diff
@@ -269,19 +268,19 @@ export default class ElectronPlatform extends VectorBasePlatform {
...
         // doesn't fire the before-quit event so the main process needs to know
         // it should exit.
-        electron.send('install_update');
+        window.electron.send('install_update');
     }
 
     public getDefaultDeviceDisplayName(): string {
         const brand = SdkConfig.get().brand;
-        return _t('%(brand)s Desktop (%(platformName)s)', {
+        return _t('%(brand)s Desktop: %(platformName)s', {
```
Comment (t3chguy) [inline src/vector/platform/ElectronPlatform.tsx:283]: Seems inconsistent to use `on X` on Web, but `: X` on Desktop

Comment (kerryarchibald) [reply src/vector/platform/ElectronPlatform.tsx:283]: That was the product decision

**Review by t3chguy — APPROVED** [review]

**Pull request merged**